src/main/java/it/gov/pagopa/pu/workflow/wf/pagopa/send/SendNotificationWFClient.java

```diff
@@ -34,7 +34,7 @@ public String sendNotificationProcess(String sendNotificationId) {
     return workflowId;
   }
 
-  public String sendNotificationDateRetrieve(String sendNotificationId) {
+  public String scheduleSendNotificationDateRetrieve(String sendNotificationId) {
```

Comment:
this is not a schedule! here you are calling it right now!

see the following for an example of WF scheduling (watch the use of workflowService#buildWorkflowStubScheduled method)
it.gov.pagopa.pu.workflow.wf.debtposition.expirationdp.CheckDebtPositionExpirationWfClientImpl#scheduleNextCheckDpExpiration


ps: anyway, let's not remove this method, and revert its name, we are interested to keep an API to execute it on-demand